source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutorObservesScriptObserverBackoffStrategy.cs

```diff
@@ -30,7 +31,11 @@ public async Task TheScriptObserverBackoffShouldBeRespected(TentacleConfiguratio
 
             var (_, logs) = await clientTentacle.TentacleClient.ExecuteScript(startScriptCommand, CancellationToken);
 
-            scriptServiceV2CallCounts.GetStatusCallCountStarted.Should().BeLessThan(3);
+            tracingStats.For(nameof(IAsyncClientScriptServiceV2.GetStatusAsync)).Started
```

Comment:
Turns out this test wasn't testing the `ScriptServiceV1` as there was no `CountCalls` so it was always returning `0`, which isn't a great test :)

I've updated this test to make sure there GetStatus call's are greater than 0 and it all passes :)